### PR TITLE
[Fix bug 1254083] Change profile settings nav menu to use bootstrap's

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
@johngian r?

File: mozillians/static/mozillians/js/profile_edit.js [hunk 1]
```diff
@@ -13,41 +13,26 @@ $(function() {
...
-        $('.settings-tab').hide();
-        $('.settings-nav > li').removeClass('active');
-        $(this).parent('li').addClass('active');
-        var content = $(this).attr('href');
-        $(content + '-tab').show();
-        location.hash = content;
-    });
     if (screen.width < mobile_breakdown) {
-        $('.settings-tab').show();
+        //$('.settings-tab').show();
```
Comment (ppapadeas) [inline mozillians/static/mozillians/js/profile_edit.js]: Remove commented coded.

**Comment by ppapadeas** [comment]
r+ wc

File: mozillians/jinja2/phonebook/edit_profile.html [hunk 2]
```diff
@@ -29,22 +29,34 @@
   {% if profile.is_complete %}
     {# This is the full profile form for those who have completed registration. #}
     <div class="row">
-      <div class="col-md-3 hidden-xs hidden-sm">
+      <div class="col-md-3 hidden-xs hidden-sm" id="settings-nav">
         <div class="panel panel-default">
           <div class="panel-heading">{{ _('Settings') }}</div>
-          <ul class="list-group settings-nav">
-            <li class="list-group-item" id="profile-li"><a href="#profile">{{ _('Profile') }}</a></li>
```
Comment (johngian) [inline mozillians/jinja2/phonebook/edit_profile.html]: Please ensure that we are not using it somewhere else and remove stale references to ``list-group-item`` in ``main.less``.

File: mozillians/static/mozillians/js/profile_edit.js [hunk 3]
```diff
@@ -13,41 +13,26 @@ $(function() {
     $('head').append(favicon);
 
     // Tabs
-    $('.settings-tab').hide();
     var uri = URI(location.href);
     var next_section = uri.query(true).next;
-    var content = uri.hash();
+    var content = window.location.hash;
```
Comment (johngian) [inline mozillians/static/mozillians/js/profile_edit.js]: Why are you using ``window.location.hash`` instead of ``uri.hash``.

**Comment by comzeradd** [comment]
@johngian PR updated. Could you have another look?

**Comment by johngian** [comment]
On mobile view (360x640) ``#get_involved.content`` is misaligned to the left.

**Comment by johngian** [comment]
r+wc from me. I suggest @akatsoulas takes another look to doublecheck that high dpi works well.

**Comment by comzeradd** [comment]
Rebased to master

File: mozillians/jinja2/phonebook/includes/profile_edit_location.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
-<form class="edit-profile" method="POST" action="{{ url('phonebook:profile_edit') }}?next=mylocation">
-  {% csrf_token %}
-  <div id="mylocation-tab" class="settings-tab">
+<div role="tabpanel" class="tab-pane fade" id="mylocation">
+  <form class="edit-profile" method="POST" action="{{ url('phonebook:profile_edit') }}?next=mylocation">
```
Comment (akatsoulas) [inline mozillians/jinja2/phonebook/includes/profile_edit_location.html]: Please change the order at the end of file too (end of form before div)

**Pull request merged**